.travis.yml

```diff
@@ -6,16 +6,17 @@ python:
   - '3.6'
 
 env:
-  - DJANGO=true
-  - SIMPLE_SETTINGS=true
+  - USE_DJANGO=true
+  - USE_SIMPLE_SETTINGS=true
+  -
```

Comment:
What do you think about adding some value to env, like `NO_THIRD_PARTY_LIBS=true` to more declarative configuration.

Reply:
Great!